xcp_d/data/executive_summary_templates/report_brainsprite_template.html

```diff
@@ -0,0 +1,51 @@
+<section id="{{ image_type }}">
+    <div class="w3-container">
+        <div class="w3-cell w3-left label3"><h6>BrainSprite Viewer: {{ image_type }}</h6></div>
+        <div class="w3-cell w3-right">
+            {{ pngs_button }}
+        </div>
+    </div>
+    <div class="w3-container">
+        <div w3-row w3-hide-small>
+            <canvas id="{{ image_type }}-viewer" style="max-width: 100%">
+            <img id="{{ image_type }}-spriteImg" class="hidden" src="{{ mosaic_path }}">
+        </div>
+    </div>
+</section>
+
+<script>
+    $( window ).load(function() {
+        var brain{{ image_type }} = brainsprite({
+            canvas: "{{ image_type }}-viewer",
+            sprite: "{{ image_type }}-spriteImg",
+            nbSlice: { 'Y':218 , 'Z':218 },
+            flagCoordinates: true,
+        });
+    });
+</script>
```

Comment:
@mattcieslak do you know if this will work with Jinja? My linter complains about trying to include templating code in the script HTML.

Reply:
Seems to work just fine.